tests/cephfs/cephfs_utilsV1.py

```diff
@@ -0,0 +1,245 @@
+            fuse_cmd = f"ceph-fuse -n client.{kwargs.get('new_client_hostname', client.node.hostname)} {mounting_dir} "
+            if kwargs.get("extra_params"):
+                fuse_cmd += f"{kwargs.get('extra_params')}"
+            client.exec_command(sudo=True, cmd=fuse_cmd)
+            out, rc = client.exec_command(cmd="mount")
+            mount_output = out.read().decode().rstrip("\n")
+            mount_output = mount_output.split()
+            log.info("Checking if fuse mount is is passed of failed:")
+            assert mounting_dir.rstrip("/") in mount_output, "Fuse mount failed"
+
+    def kernel_mountV1(self, kernel_clients, mounting_dir, mon_node_ip, **kwargs):
+        """
+        Mounts the drive using kernel mount
+        Args:
+            kernel_clients:
+            mounting_dir:
+            mon_node_ip:
+            **kwargs:
+
+        Returns:
+        """
+        for client in kernel_clients:
+            log.info("Creating mounting dir:")
+            client.exec_command(sudo=True, cmd="mkdir %s" % mounting_dir)
+            out, rc = client.exec_command(
+                sudo=True,
+                cmd=f"ceph auth get-key client.{kwargs.get('new_client_hostname', client.node.hostname)} -o "
+                f"/etc/ceph/{kwargs.get('new_client_hostname', client.node.hostname)}.secret",
+            )
+
+            kernel_cmd = (
+                f"mount -t ceph {mon_node_ip}:/ {mounting_dir} "
+                f"-o name={kwargs.get('new_client_hostname', client.node.hostname)},"
+                f"secretfile=/etc/ceph/{kwargs.get('new_client_hostname', client.node.hostname)}.secret"
+            )
+
+            if kwargs.get("extra_params"):
+                kernel_cmd += f"{kwargs.get('extra_params')}"
+            client.exec_command(
+                sudo=True,
+                cmd=kernel_cmd,
+            )
+            out, rc = client.exec_command(cmd="mount")
+            mount_output = out.read().decode()
+            mount_output = mount_output.split()
+            log.info("Checking if kernel mount is is passed of failed:")
+            assert mounting_dir.rstrip("/") in mount_output, "Kernel mount failed"
+
+    def get_mon_node_ips(self):
+        """
+
+        Returns:
+
+        """
+        return [node.ip_address for node in self.ceph_cluster.get_nodes(role="mon")]
+
+    @staticmethod
+    def client_clean_up(fuse_clients, kernel_clients, mounting_dir, *args, **kwargs):
+        clients = fuse_clients + kernel_clients
+        for client in clients:
+            log.info("Removing files:")
+            client.exec_command(
+                sudo=True,
+                cmd=f"find {mounting_dir} -type f -delete",
+                long_running=True,
+                timeout=3600,
+            )
+            client.exec_command(
+                sudo=True,
+                cmd="rm -rf %s*" % mounting_dir,
+                long_running=True,
+                timeout=3600,
+            )
```

Comment:
This should suffice for cleaning up the directory. i.e. `cmd=f"rm -rf  {mounting_dir}/*"`